### DISCUSSION TITLE
How to compile this plug-in?

### DISCUSSION THREAD

**Discussion by 2020AD** [grafana/grafana-infinity-datasource#360] [Q&A] [answered]
How to compile this plug-in? What commands do the front end and the back end need to execute to make the plug-in effective？

**Comment by yesoreyeram** [comment, marked as answer]
'yarn dev' for the frontend. And 'mage -v' for the backend compilation. Read more in CONTRIBUTING.md file

**Reply by 2020AD** [reply]
okey，thank you
